modules/stopping-an-active-pipeline-run.adoc

```diff
@@ -19,9 +19,10 @@ endif::[]
 * An active pipeline run is currently executing.
 
 .Procedure   
-. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Runs*.
-. On the *Runs* page, select the *project* that contains the pipeline whose active run you want to stop.
-. In the *Active* tab, click the action menu (*&#8942;*) beside the active run that you want to delete and click *Stop*.
+. From the {productname-short} dashboard, click *Experiments* -> *Experiments and runs*.
+. On the *Experiments* page, select the *project* that contains the pipeline experiment for the active run that you want to stop.
+. From the list of pipeline experiments, click the pipeline experiment that contains the run that you want to stop. 
+. In the *Runs* tab, click the action menu (*&#8942;*) beside the active run that you want to stop and click *Stop*.
```

Comment:
Suggest updating to` 'stop, and then click *Stop*.'`

Reply:
Changed to your suggestion.